pom.xml

```diff
@@ -112,12 +118,6 @@
             <artifactId>guava</artifactId>
             <version>19.0</version>
         </dependency>
-        <!--json stuff-->
-        <dependency>
-            <groupId>com.googlecode.json-simple</groupId>
-            <artifactId>json-simple</artifactId>
-            <version>1.1.1</version>
-        </dependency>
```

Comment:
Hmm, are you sure this isn't needed? Not sure if this is leftover from me copying the pom for GriefPrevention long ago, or if it's still needed. From what I recall, I observe projects building if these exist in either the target directory or maven cache.

Reply:
I don't see that we are using this json library anywhere. It was also removed from GriefPrevention in later versions.